kong/plugins/galileo/buffer.lua

```diff
@@ -111,7 +112,7 @@ _send = function(premature, self, to_send)
   local client = http.new()
   client:set_timeout(self.connection_timeout)
 
-  local ok, err = client:connect(self.host, self.port)
+  local ok, err = connect(client, self.host, self.port)
```

Comment:
We could implement this in our `globalpatches` module now, to avoid diverging from the tcpsock regular API.